src/components/BlogSection/BlogPost/BlogPost.js

```diff
@@ -0,0 +1,44 @@
+import Button from '../../UI/Button';
+import PropTypes from 'prop-types';
+
+const BlogPost = ({ title, author, date, content, imageUrl, id, onClick }) => {
+  const shortTitle = title.length >= 35 ? title.slice(0, 35) + ' ...' : title;
```

Comment:
We can truncate text with CSS :)

Reply:
nice feature, I didn't use it before. I wonder that do I use best solution. We can take a look for it on our next meeting.